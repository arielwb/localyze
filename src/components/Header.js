import React from 'react'

export const Header = ({ id, children }) => {
  return (
    <>
      <h1>Challenge {id}</h1>
      <p>{children}</p>
    </>
  )
}
