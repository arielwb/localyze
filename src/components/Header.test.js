import React from 'react'
import renderer from 'react-test-renderer'
import { Header } from './Header'

describe('Header', () => {
  it('should match the snapshot', () => {
    const tree = renderer
      .create(<Header id={1}>A beautiful test</Header>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
