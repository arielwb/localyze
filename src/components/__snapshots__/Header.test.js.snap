// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
Array [
  <h1>
    Challenge 
    1
  </h1>,
  <p>
    A beautiful test
  </p>,
]
`;
